.github/workflows/e2e-cypress.yml

```diff
@@ -62,11 +62,8 @@ jobs:
         run: yarn cypress run --spec cypress/e2e/parallel-${{ matrix.group }}/*
         working-directory: ./commercial
 
-      # Only capture videos and screenshots on failure
-      # In order to minimize unused artifacts
-
       - uses: actions/upload-artifact@v2
-        if: failure()
+        if: always()
```

Comment:
I'll leave this here in case anyone wants to override `video: true` in the Cypress config of their branch to do debugging.